validator-api/app.py

```diff
@@ -879,7 +883,7 @@ async def run_server():
             server_task,
             resync_metagraph(),
             cache_max_focus_tao(),
-            resync_dataset(),
+            # resync_dataset(),
```

Comment:
Need to remember not to leave in little mistakes like this. Can maybe use a flag to automatically disable it when in dev mode so that we dont have to remember to comment it out and back in